app/views/devise/registrations/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="jumbotron text-center">
```

Comment:
question: when does this page show up?

Reply:
soon after the signup or login _/users/_ page